test/activation_based/test_flex_sn_inductor.py

```
@@ -0,0 +1,1294 @@
+    x = torch.empty(0, 8)
+    v0 = torch.zeros(8)
+
+    with pytest.raises(ValueError, match="T == 0"):
+        flex_sn_scan(_lif_core, 1, 1, 1, x, v0)
+
+    with pytest.raises(ValueError, match="T == 0"):
+        flexsn_hop.eager_scan_final_state(_lif_core, 1, 1, 1, x, v0)
+
+
+def test_kernel_names_include_graph_fingerprint():
+    core_a = lambda x: x + 1
```

Ruff (0.15.11)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `core_a` as a `def`

(E731)